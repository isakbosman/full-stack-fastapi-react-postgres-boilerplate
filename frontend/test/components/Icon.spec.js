import React from 'react';

import Icon from 'components/Icon';

describe('Icon', () => {
  const wrapper = mount(<Icon name="check" />);

  it('should render properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
