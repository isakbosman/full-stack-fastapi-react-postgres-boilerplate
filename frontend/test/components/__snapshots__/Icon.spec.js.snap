// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon should render properly 1`] = `
<IconWrapper
  src="/media/icons/check.svg"
  width={20}
/>
`;
